src/locales/fr.json

```
@@ -694,6 +694,18 @@
     "generic": "Mes factures"
   },
 
+  "PersonalInfo": {
+    "birthcity": "Commune de naissance",
+    "nationality": "Nationalité",
+    "save-cta": "Sauver mon profil",
```

```suggestion
    "save-cta": "Sauvegarder mon profil",
```

J'aurais plus mis ca non?